worker-batch-ingestion-plugin/src/main/java/com/github/cafdataprocessing/worker/ingestion/IngestionBatchWorkerPlugin.java

```diff
@@ -64,6 +66,8 @@ public final class IngestionBatchWorkerPlugin implements BatchWorkerPlugin
 {
     private final ObjectMapper mapper;
     private final BatchPathProvider fileSystemProvider;
+    private final String validationFile;
+    private final FieldValidator fieldValidator;
```

Comment:
Rather than storing both of these and creating a dummy unused field validator when the file is not specified, create an interface for your field validator to implement, and then create a singleton implementation of it that does nothing.  You can then set the field validator to `NullFieldValidator.INSTANCE` if the validation file is not specified.

    public interface FieldValidatorInterface
    {
        DocumentWorkerDocument validate(DocumentWorkerDocument document);
    }

    public enum NullFieldValidator implements FieldValidatorInterface
    {
        INSTANCE;

        @Override
        public DocumentWorkerDocument validate(final DocumentWorkerDocument document)
        {
            return document;
        }

        @Override
        public String toString()
        {
            return "<null field validator>";
        }
    }

```suggestion
    private final FieldValidatorInterface fieldValidator;
```